CMakeLists.txt

```diff
@@ -2398,7 +2398,10 @@ if(BROADCAST)
   elseif(Shoutidjc_FOUND AND Shoutidjc_VERSION VERSION_LESS 2.4.6)
       message(STATUS "Installed libshout version: ${Shout_VERSION} is suffering from bug lp1913579")
   endif()
-  if(NOT Shsoutidjc_FOUND OR Shoutidjc_VERSION VERSION_LESS 2.4.6)
```

Comment:
I'm pretty sure I pointed this out during review, I wonder how this got lost?

Reply:
probably a merge conflict resolution